### PR TITLE
Merged dev with release-1.2.0 branch

### PR DESCRIPTION
**Pull request by mayuradesh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,49 @@
-[![Build Status](https://travis-ci.org/mosip/pre-registration.svg?branch=master)](https://travis-ci.org/mosip/pre-registration)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=mosip_pre-registration&metric=alert_status)](https://sonarcloud.io/dashboard?id=mosip_pre-registration)
-[![Join the chat at https://gitter.im/mosip-community/Pre-registration](https://badges.gitter.im/mosip-community/Pre-registration.svg)](https://gitter.im/mosip-community/Pre-registration?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Maven Package upon a push](https://github.com/mosip/pre-registration/actions/workflows/push_trigger.yml/badge.svg?branch=release-1.2.0)](https://github.com/mosip/pre-registration/actions/workflows/push_trigger.yml)
```
Comment (gsasikumar) [inline README.md]: Branch name has to be changed to develop

File: README.md [hunk 2]
```diff
@@ -1,58 +1,49 @@
...
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=release-1.2.0&project=mosip_pre-registration&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.2.0&id=mosip_pre-registration)
```
Comment (gsasikumar) [inline README.md]: branch name has to be changed to develop

**Review by gsasikumar — CHANGES_REQUESTED** [review]

**Review by ckm007 — COMMENTED** [review]
Can see difference in db scripts and all... can we confirm ifthe same changes are not part of develop... as they should have everything commons except Readme, .github files and few plugins additions

**Comment by mayuradesh** [comment]
> Can see difference in db scripts and all... can we confirm ifthe same changes are not part of develop... as they should have everything commons except Readme, .github files and few plugins additions

As discussed with Mandeep, all the new db changes have to be moved from "release-1.2.0" branch to "develop".

File: README.md (hunk 2 above)
Comment (mayuradesh) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (mayuradesh) [reply README.md]: done

**Pull request merged**